aws/rust-runtime/aws-config/src/web_identity_token.rs

```diff
@@ -193,6 +199,32 @@ impl Builder {
         self
     }
 
+
+    /// Set an IAM policy in JSON format that you want to use as an inline session policy.
+    ///
+    /// This parameter is optional
+    /// For more information, see
+    /// [policy](aws_sdk_sts::input::assume_role_input::Builder::policy_arns)
+    pub fn policy(mut self, policy: impl Into<String>) -> Self {
+        self.policy = Some(policy.into());
+        self
+    }
+
+    /// Set the Amazon Resource Names (ARNs) of the IAM managed policies that you want to use as managed session policies.
+    ///
+    /// This parameter is optional.
+    /// For more information, see
+    /// [policy_arns](aws_sdk_sts::input::assume_role_input::Builder::policy_arns)
+    pub fn policy_arns(mut self, policy_arns:  Vec<String>) -> Self {
```

Comment:
Is there a reason this method takes `Vec<String>` instead of `Vec<PolicyDescriptorType>` (to make it consistent with `AssumeRoleProviderBuilder::policy_arns`)?

Also, please make sure to install [pre-commit hook](https://github.com/smithy-lang/smithy-rs?tab=readme-ov-file#development) to run `cargo fmt` as part of commit.

Reply:
Thanks for the feedback. I intentionally made the parameter `Vec<String>` for consistency. In the PR I mentioned above it was originally `Vec<PolicyDescriptorType>` however in the current doc it is a `Vec<String>` https://docs.rs/aws-config/latest/aws_config/sts/struct.AssumeRoleProviderBuilder.html#method.policy_arns

Reply:
Actually, you're right. By using `String` it helps avoid exposing `PolicyDescriptorType` from a public API (it's a good practice to avoid exposing generated types).

Reply:
@ysaito1001 Fixed the linting issues pointed out by the precommit hook including the cargo fmt